toolset/setup/linux/frameworks/cppcms.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+fw_installed cppcms && return 0
+
+#http://cppcms.com/wikipp/en/page/cppcms_1x_build
+
+#note the -rc1 in url
+VERSION=1.1.1
+BACKNAME=cppcms
+CPPCMS_HOME=$IROOT/$BACKNAME-$VERSION
+
+sudo apt-get update -qq
+sudo apt-get install -y cmake libpcre3-dev zlib1g-dev libgcrypt11-dev libicu-dev python
+
+fw_get -o $BACKNAME-$VERSION.tar.bz2 https://download.sourceforge.net/project/cppcms/$BACKNAME/$VERSION-rc1/$BACKNAME-$VERSION.tar.bz2
+fw_untar $BACKNAME-$VERSION.tar.bz2
+
+cd $BACKNAME-$VERSION
+rm -rf build
+mkdir build
+cd build
+cmake ..
+
+make
+#make test
+sudo make install
```

Comment:
Same comment here about no `sudo` and using `$IROOT`.

See [this](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/webservers/openresty.sh) for an example of how others are doing it.